ci/create_database.py

```diff
@@ -89,79 +85,54 @@ async def create_database():
             assert len(rows) == 1
             return
 
-    with open(create_database_config['admin_password_file'], encoding='utf-8') as f:
-        admin_password = f.read()
-
-    with open(create_database_config['user_password_file'], encoding='utf-8') as f:
-        user_password = f.read()
-
-    admin_exists = await db.execute_and_fetchone("SELECT user FROM mysql.user WHERE user=%s", (admin_username,))
-    admin_exists = admin_exists and admin_exists.get('user') == admin_username
-
-    user_exists = await db.execute_and_fetchone("SELECT user FROM mysql.user WHERE user=%s", (user_username,))
-    user_exists = user_exists and user_exists.get('user') == user_username
-
-    create_admin_or_alter_password = (
-        f"CREATE USER '{admin_username}'@'%' IDENTIFIED BY '{admin_password}';"
-        if not admin_exists
-        else "ALTER USER '{admin_username}'@'%' IDENTIFIED BY '{admin_password}';"
-    )
+    async def create_user_idempotent(admin_or_user, mysql_username, mysql_password):
+        existing_user = await db.execute_and_fetchone('SELECT user FROM mysql.user WHERE user=%s', (mysql_username,))
+        if existing_user is not None and existing_user.get('user') == mysql_username:
```

Comment:
the second condition is always true if the first condition is true, right? I'm fine making it an assertion below.

Reply:
Ya true I was just shifting code around. Just going to delete this check and `SELECT 1`.